src/lib/connectors/repos/events.js

```diff
@@ -34,7 +34,17 @@ const getMostRecentReturnsInvitationByLicence = async licenceRef => {
   return bookshelf.knex.raw(queries.getMostRecentReturnInvitation, { licenceRef });
 };
 
+const getKPIReturnsMonthlyData = async () => {
```

Comment:
Not sure you need `async` here because the `raw` function will already be returning a promise

Reply:
thank you

Reply:
Hi Mark your suggestion doesn't work because of the subsequent lines extracting the rows and converting to camelCase.

Reply:
done thanks